src/renderTopLanguages.js

```diff
@@ -7,7 +7,7 @@ const createProgressNode = ({ width, color, name, progress }) => {
   const progressPercentage = clampValue(progress, 2, 100);
 
   return `
-    <text data-testid="lang-name" x="2" y="15" class="lang-name">${name}</text>
+    <text data-testid="lang-name" x="2" y="15" class="lang-name lang-name-${cleanLangName(name)}">${name}</text>
```

Comment:
Any particular reasons why you are giving each lang a unique CSS class? or just because you wanted to test them? maybe we can just use queryAllByTestId to do that instead.

Reply:
This was just for testing. I'll update the tests to use `queryAllByTestId` instead.